hello.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+const CLONED_PROJECTS_DIR string = "/tmp/cloned-projects"
+
+type Input struct {
+	CloneUrl string `json:"clone_url"`
+	Size     int
+}
+
+type File struct {
+	Name string
+	Size int
+}
+
+type Scan struct {
+	Total int
+	Files []File
+}
+
+func main() {
+	input, err := readInputFile()
+	if err != nil {
+		fmt.Println("Error with read input file:", err)
+		return
+	}
+
+	if err := gitClone(input.CloneUrl); err != nil {
+		fmt.Println("Error with clone repo:", err)
+		return
+	}
+
+	scan := scanRepoFiles(CLONED_PROJECTS_DIR, input.Size*1000)
+
+	if err := writeOutputFile(scan); err != nil {
+		fmt.Println("Error with write output file:", err)
+	}
+
+	path, err := filepath.Abs("output.json")
+	if err != nil {
+		fmt.Println("Error getting absolute path:", err)
+	}
+	fmt.Printf("%s", path)
```

Comment:
```suggestion
	fmt.Println(path)
```
Or you can add more text and then do:
```
	fmt.Printf("result path - %s\n")
```

FYI, `fmt.Printf` doesn't append a line break, so every `fmt.Printf` statement has to end with `\n`